docs/nodes/nodes-library/nodes/Excel/README.md

```diff
@@ -0,0 +1,46 @@
+---
+permalink: /nodes/n8n-nodes-base.microsoftExcel
+---
+
+# Excel
```

Comment:
I see that the node is named 'Microsoft Excel'.

<img width="158" alt="Screenshot 2020-07-29 at 08 26 06" src="https://user-images.githubusercontent.com/7481165/88764380-2926aa00-d175-11ea-830d-bcc0d03359aa.png">